projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -46,28 +47,47 @@ class Notebook:
     Other cells are ignored.
     """
 
-    def __init__(self, file_path: Path):
+    @staticmethod
+    def register_notebook_steps(file_path: Path, context: JobContext):
         try:
-            self.sql_and_run_cells = []
-            self.python_helper_cells = []
-            self.file_path = file_path
+            python_module = imp.new_module("notebook")
```

Comment:
I see this warning?

`the imp module is deprecated in favour of importlib and slated for removal in Python 3.12; see the module's documentation for alternative uses `

Can you use importlib instead? 
I know nothing about those libraries.

Reply:
Yes, I know currently in vdk core we use imp, too. If I cannot find any good alternatives I will leave it like that for now, if I find something good, will change it

Reply:
I changed it with importlib